src/lib/support/Span.h

```diff
@@ -390,4 +390,18 @@ inline CHIP_ERROR CopyCharSpanToMutableCharSpan(CharSpan cspan_to_copy, MutableC
     return CHIP_NO_ERROR;
 }
 
+/**
+ * Copies a CharSpan into a MutableCharSpan.
+ * If the cspan_to_copy does not if in out_buf, cspan_to_copy in truncated to fit in out_buf.
+ * @param cspan_to_copy The CharSpan to copy.
+ * @param out_buf The MutableCharSpan in which cspan_to_copy is to be copied.
+ */
+inline void CopyCharSpanToMutableCharSpanTruncate(CharSpan cspan_to_copy, MutableCharSpan & out_buf)
```

Comment:
Suggest renames to span_to_copy, out_span

Reply:
Accepted. See [this commit](https://github.com/project-chip/connectedhomeip/pull/35360/commits/04161757bd2964d7c8ea1599f2c1256a5292ac2b).